docs/docs/index.md

```diff
@@ -63,21 +65,20 @@ probably be named something much less useful!
 ### Predict Using Concise Output Format
 
 If you just want to know the most likely animal in each video, the
-`--output_class_names` flag is useful. In this case, the final output as well as the resulting `output.csv`
+`--output-class-names` flag is useful. In this case, the final output as well as the resulting `output.csv`
 are simplified to show the _most probable_ animal in each video:
 
 ```console
-$ zamba predict vids_to_classify/ --output_class_names
+$ zamba predict --data-dir vids_to_classify/ --output_class_names
 ...
-blank2.mp4                blank
+blank.mp4                 blank
+chimp.mp4     chimpanzee_bonobo
 eleph.mp4              elephant
-blank1.mp4                blank
-ungulate.mp4     large ungulate
-small-cat.mp4         small cat
+leopard.mp4             leopard
 ```
 
 **NOTE: `zamba` needs to download the "weights" files for the neural networks
-that it uses to make predictions. On first run it will download ~1GB of files
+that it uses to make predictions. On first run it will download ~1GB <!-- TODO: check size><!--> of files
```

Comment:
FYI I checked the size of the checkpoints I believe we're using for time distributed, slowfast, and european, and together they still add up to almost exactly 1 GB

Reply:
you'll just get the weights for whichever model you choose, not all three